src/test/kotlin/com/personio/synthetics/e2e/E2EMultiStepApiTest.kt

```diff
@@ -80,6 +80,8 @@ class E2EMultiStepApiTest {
                         assertions {
                             statusCode(200)
                             bodyContainsJsonPath("\$.success", "true")
+                            // regex to match "t{single-char}u{single-char}"
```

Comment:
Let's remove this comment considering that the example is clear and the function contains explaination